java/ql/src/Security/CWE/CWE-601/UrlRedirect.ql

```diff
@@ -13,14 +13,19 @@
 import java
 import semmle.code.java.dataflow.FlowSources
 import semmle.code.java.security.UrlRedirect
+import semmle.code.java.dataflow.ExternalFlow
 import DataFlow::PathGraph
 
 class UrlRedirectConfig extends TaintTracking::Configuration {
   UrlRedirectConfig() { this = "UrlRedirectConfig" }
 
   override predicate isSource(DataFlow::Node source) { source instanceof RemoteFlowSource }
 
-  override predicate isSink(DataFlow::Node sink) { sink instanceof UrlRedirectSink }
+  override predicate isSink(DataFlow::Node sink) {
+    sink instanceof UrlRedirectSink
+    or
+    sinkNode(sink, "url-redirect")
```

Comment:
Add this as a private subclass of `UrlRedirectSink` in `UrlRedirect.qll` instead.